build.py

```diff
@@ -24,7 +32,27 @@
     # Number of available CPUs as a fall-back (may be None)
     NUM_CPUS = os.cpu_count()
 
-def print_usage():
+
+class RawHelpFormatter(argparse.HelpFormatter):
+    """Preserve newlines and spacing."""
+    def _fill_text(self, text, width, indent):
+        return ''.join([indent + line for line in text.splitlines(True)])
```

Comment:
We used to generally use `""` rather than `''`, although I'm not sure if this ship has sailed and we're now mixing these freely. (I think in the past I tried to be consistent about this at least in the driver and translator, but not for example in the tests. But we may have given up on consistency for this already.)

Reply:
There are a couple of `''` in the driver, mostly in the portfolio definitions. We also use them for output like `print("Expected ')'")` but this is a different situation. Overall, it is pretty consistently `""` but there are a couple of cases that should be changed besides this one (this particular use is copied from existing code in the driver). I'll only change the parts that the issue touches anyway to avoid increasing the diff.